timesketch/api/v1/resources/event.py

```diff
@@ -623,8 +650,12 @@ def post(self, sketch_id):
                     sketch.id,
                     current_user.id,
                     form.annotation.data,
-                    toggle=toggle)
+                    toggle=toggle
+                )
 
+                if current_search_node:
+                    current_search_node.add_label(form.annotation.data)
```

Comment:
why adding the label itself as well, not just the notion that a label has been added? (which is done in the line below)... similar to what is done in comments

Reply:
Yeah, I have been going back and forth on this one. But I think you are right, we should skip it. The idea was to do some UI work to show what labels was added but I think this is too early. We can always add in later if needed. 

I'll remove it for now.